v1-2/learn/how-to-test-ballerina-code.md

```diff
@@ -403,3 +404,74 @@ function testService() {
     }
 }
 ```
+
+## Function Mocks
+Testerina provides the functionality to mock a function in a different thrird-part module with your own ballerina function, which will help you test your ballerina module independently.
+
+#### @test:Mock {}
+
+The function specified with the annotation `@test:Mock {}` will be considered as a mock function that gets triggered everytime the original function is called. The original function that will be mocked should be defined using the annotation parameters
+
+##### Annotation Value Fields
+
+`moduleName : "<moduleName>"` : Name of the module where the function to be mocked resides in. If the function is within the same module, this can be left blank or "." (No module) can be passed. If the function is in a different module, but within the same project, just passing the module name will suffice. For functions in completely seperate modules, the fully qualified module name must be passed, which includes the `orgName` and the `version`. ie. `orgName/module:version`. For native function, the ballerina module needs to be specified. 
```

Comment:
```suggestion
`moduleName : "<moduleName>"` : Name of the module where the function to be mocked resides in. If the function is within the same module, this can be left blank or "." (no module) can be passed. If the function is in a different module, but within the same project, just passing the module name will suffice. For functions in completely separate modules, the fully-qualified module name must be passed, which includes the `orgName` and the `version`. i.e., `orgName/module:version`. For native functions, the Ballerina module needs to be specified. 
```

Reply:
Here the version is optional right ?

Reply:
When giving the fully qualified name, the version is also required in this case. Because it does all checks with the fully qualified module name.